ClosedXML/Excel/Ranges/XLRangeAddress.cs

```diff
@@ -1,12 +1,31 @@
 using ClosedXML.Extensions;
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
 
 namespace ClosedXML.Excel
 {
-    internal class XLRangeAddress : IXLRangeAddress
+    internal struct XLRangeAddress : IXLRangeAddress, IEquatable<XLRangeAddress>
     {
+        #region Static members
+
+        public static XLRangeAddress EntireColumn(XLWorksheet worksheet, int column)
+        {
+            return new XLRangeAddress(
+                new XLAddress(worksheet, 1, column, false, false),
+                new XLAddress(worksheet, XLHelper.MaxRowNumber, column, false, false));
+        }
+
+        public static XLRangeAddress EntireRow(XLWorksheet worksheet, int row)
+        {
+            return new XLRangeAddress(
+                new XLAddress(worksheet, row, 1, false, false),
+                new XLAddress(worksheet, row, XLHelper.MaxColumnNumber, false, false));
+        }
+
+        #endregion Static members
+
         #region Private fields
```

Comment:
Remove the entire `Private fields` region, including `_firstAddress` and `_lastAddress`. Implement autoproperty in `FirstAddress` and `LastAddress`